CoroutineTemplate/build.gradle.kts

```diff
@@ -35,8 +35,12 @@ tasks.withType<io.gitlab.arturbosch.detekt.Detekt>().configureEach {
 }
 
 detekt {
-    toolVersion = Versions.DETEKT_VERSION
-
+    source = files(
```

Comment:
Seems like we were using keyword `input` instead of `source` [before](https://github.com/nimblehq/android-templates/pull/132/files#diff-11634f378c1a3f8859e26878fd36fb4b6ae2c52f04dae789a1202f8620c20722L73). Is there any difference? 🙏

Reply:
@Tuubz I mentioned it in PR description

<img width="796" alt="image" src="https://user-images.githubusercontent.com/16315358/163916922-5b7559e7-1816-48bd-9f74-a18ba16f000d.png">